src/components/LendModals/index.tsx

```diff
@@ -488,7 +491,11 @@ export const QuickModalContent: React.FC<QuickModalContentProps> = ({
             <Box className='flex items-center'>
               <CurrencyLogo
                 currency={getPoolAssetToken(asset, chainId)}
-                withoutBg={asset.underlyingName.includes('LP')}
+                withoutBg={
+                  asset.underlyingName.includes('LP') ||
+                  asset.underlyingSymbol.includes('am') ||
+                  asset.underlyingSymbol.includes('moo')

```

Comment:
Thought we weren't supporting moo

Reply:
There are some LP assets including `moo` in symbol.